server/app/Http/Controllers/GroupController.php

```diff
@@ -37,7 +38,15 @@ public function store(StoreGroupRequest $request)
         $group->status = Group::STATUS_PENDING;
         $group->save();
         $group->addMember(Auth::user()->id, Group::ROLE_OWNER);
-        Event::fire(new GroupCreated($group));
+
+        switch ($group->individual) {
+        case true:
+            Event::fire(new IndependentSponsorRequest(Auth::user()));
```

Comment:
I don't think we need a separate event for this case. At an application level, the event that is occurring is a group is being created. If we want different messaging based on what kind of group it is, we can do that in the group create event listener that sends the notification. We lose the separate notification preference for independent sponsor groups vs other groups, but I don't think that's a real issue here.